terraform/deployments/govuk-publishing-platform/app_router_api.tf

```diff
@@ -27,6 +32,7 @@ module "router_api" {
   source = "../../modules/app"
 
   service_name                     = "router-api"
+  backend_virtual_service_names    = local.router_api_defaults.backend_services # TODO what services does Router API use?
```

Comment:
I guess it should probably have Router in its services (although I'm not sure we've really hooked that up properly). I don't think it talks to any other services, other than maybe signon which you've already got.